### DISCUSSION TITLE
Help with deploying EDC locally.

### DISCUSSION THREAD

**Discussion by cipherscribe7** [eclipse-tractusx/tractusx-edc#1240] [General]
Could someone share the steps to build EDC from code on my local machine? list of services that needs to be running before building the EDC?

I am trying to deploy using Postgres and hashicorp vault. 

I have MIW/keycloak/postgres/vault running in local machine. But when I use the default configuration.properties to run EDC, I always get a 404 on all my requests.

**Comment by wolf4ood** [comment]
Hi @cipherscribe7 

you can check you the MXD repository which guides you to deploy two connectors and the dependencies needed

https://github.com/eclipse-tractusx/tutorial-resources

Thanks

**Reply by cipherscribe7** [reply]
Thanks for the response! I was able to get the MXD repo working. I’m trying to build from source to run one instance of EDC in each machine to do e2e testing. What am I missing?

**Reply by wolf4ood** [reply]
What are you building exactly? You can use the helm charts or the docker images in case

**Reply by cipherscribe7** [reply]
https://eclipse-tractusx.github.io/docs-kits/23.12/kits/tractusx-edc/docs/kit/operation-view/page03_local_setup_controlplane

trying to do this to be precise. running MIW kit/keycloak/postgres/vault locally and updated  those in configuration.properties.

**Comment by lgblaumeiser** [comment]
Closed as stale, please renew item if this is still an issue
